### ISSUE TITLE
Allowlist.json file example and usage

### ISSUE THREAD

**Issue by trigun539** [itzg/docker-minecraft-bedrock-server#286]
Having a hard time getting the `allowlist.json` file to work. Keep getting the user has not been invited message when connecting. Can we please add some example of how to set this up properly? Format of the `allowlist.json` and usage with the right environment variables.

Is `ALLOW_LIST_USERS` a comma separated `xuid`s,  `uuid`s? or usernames?

**Comment by itzg** [comment]
I should probably update the section heading, but this is the description and example:

https://github.com/itzg/docker-minecraft-bedrock-server#whitelist

**Comment by jonvel** [comment]
I think the wiki page that you reference might have to be updated.  Does the `allowlist.json` still use UUID's for users, or just XBox Gamertags?

**Comment by itzg** [comment]
The new section link is https://github.com/itzg/docker-minecraft-bedrock-server/tree/master#allowlist

As for `ALLOW_LIST_USERS` it is a list of Xbox gamer tags.

**Comment by trigun539** [comment]
Here what worked for me, sample `allowlist.json`

```json
[
  {
    "name": "XBOX USERNAME",
    "xuid": "xuid gotten from website below"
  },
  ....
]
```
To get xuid, look at: https://cxkes.me/xbox/xuid
Type xbox username and copy the `XUID (DEC)` given. This value is what you use in the `allowlist.json` `xuid` field.

**Comment by racecarburton** [comment]
There are ways Minecraft has automated the creation of the allowlist.json file. You just have to enter the console command "allowlist add <xbox gamertag>" and it will automatically add the xuid to the file when a player with the matching gamertag joins. It's all outlined in the "bedrock_srver_how_to.html" file as of writing this.

**Comment by itzg** [comment]
Good point @racecarburton . That could be utilized by running the commands this way https://github.com/itzg/docker-minecraft-bedrock-server/tree/master#executing-server-commands

Being able to pre-assign those users on a freshly created container is also useful, but gets into this challenge of finding those XUIDs to assign.
